pkg/policy/consumer.go

```diff
@@ -65,7 +64,7 @@ func NewConsumable(id NumericIdentity, lbls *Identity, cache *ConsumableCache) *
 		Iteration:    0,
 		Labels:       lbls,
 		Maps:         map[int]*policymap.PolicyMap{},
-		Consumers:    map[NumericIdentity]*Consumer{},
+		Consumers:    map[NumericIdentity]bool{},
 		ReverseRules: map[NumericIdentity]*Consumer{},
```

Comment:
Why is Consumers a `map[NumericIdentity]bool,` but `ReverseRules` not a `map[NumericIdentity]bool` as well?

Reply:
This is addressed by a later commit in the PR.